modules/gmc-agent/src/main/java/org/gridgain/agent/Agent.java

```diff
@@ -364,7 +366,8 @@ private void writeToMetaStorage(ManagementConfiguration cfg) {
                 if (disconnected.compareAndSet(false, true)) {
                     log.error("Lost websocket connection with server: " + curSrvUri);
 
-                    execSrvc.submit(Agent.this::connect0);
+                    if (execSrvc.getActiveCount() == 0)
```

Comment:
Please, give small comment for this condition.

Reply:
Done.